extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -121,11 +153,19 @@ export class AmpAudio extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
-    const layout = this.getLayout();
-    if (layout !== Layout.NODISPLAY) {
-      this.buildAudioElement();
+    this.buildAudioElement();
```

Comment:
I think we should still have the layout check (and return early), since you can have something be `nodisplay` that you make visible (and thus layout) using `amp-bind` at a later point. I'm not sure why you would want to do it, but it is possible.